include/manager.hpp

```diff
@@ -30,15 +31,85 @@ class Manager
      * @param[in] iFace - interface to implement.
      * @param[in] connection - Dbus Connection.
      */
-    Manager(const std::shared_ptr<boost::asio::io_context>& ioCon,
-            const std::shared_ptr<sdbusplus::asio::dbus_interface>& iFace,
-            const std::shared_ptr<sdbusplus::asio::connection>& asioConnection);
+    Manager(
+        const std::shared_ptr<boost::asio::io_context>& ioCon,
+        const std::shared_ptr<sdbusplus::asio::dbus_interface>& operationIFace,
+        const std::shared_ptr<sdbusplus::asio::connection>& asioConnection);
 
     /**
      * @brief Destructor.
      */
     ~Manager() = default;
 
+    /**
+     * @brief Write a keyword value on the given placeholder.
+     *
+     * API can be used to write keyword value either on cache/hardware/both
+     * based on users choice. By default choice is taken as to update both cache
+     * and hardware.
+     *
+     * @param[in] i_input - Input to write keyword.
+     * @param[in] i_writeOn - Write keyword to the mentioned place holder. By
+     * default write on both cache and hardware.
+     *
+     * @return Data which is written successfully.
+     */
+    types::ReturnVPD
+        writeKeyword(const types::WriteVPD i_input,
+                     uint8_t i_writeOn = constants::cacheAndHardware);
+
+    /**
+     * @brief Read keyword from the given placeholder
+     * API can be used to read keyword value either from cache/hardware/both
+     * based on users choice. By default choice is taken as to read from
+     * hardware.
+     *
+     * @param[in] i_input - Input to read keyword value.
+     * @param[in] i_readFrom - Read from the given place holder. By default read
+     * from hardware.
+     *
+     * @return Read value.
+     */
+    types::ReturnVPD readKeyword(const types::ReadVPD i_input,
+                                 uint8_t i_readFrom = constants::hardware);
+
+    /**
+     * @brief Collect single FRU VPD
+     * API can be used to perform VPD collection for the given FRU.
+     *
+     * @param[in] i_invPath - Inventory path
```

Comment:
Inventory path is Dbus path. 
We should be more specific here by using either Dbus path or EEPROM path in the doxygen.
Inventory path may not make sense to external users.

Reply:
for collectSingleFRUVPD method, we expect the user to provide Inventory path only.

Reply:
dbus_object path